db/data/regions/CO/en.yml

```diff
@@ -8,4 +8,5 @@ en:
             default: Province
             optional: Province (optional)
           errors:
-            blank: Select a province
+            blank_instructional: Select a province
+            blank_informative: Enter a province
```

Comment:
I'm guessing this is supposed to be `Province is missing`? Currently these messages seem like field validations in a form, one applicable to a drop-down and the other applicable to an open text field 🤔